### DISCUSSION TITLE
Should api be allowed to return arrays?

### DISCUSSION THREAD

**Discussion by kevwan** [zeromicro/go-zero#1047] [Ideas]
Ideas are welcome!

**Comment by vcqr** [comment]
不允许返回的理由先写一下？

**Comment by vcqr** [comment]
返回的数据类型应该是通用能力吧？
相应的规范应该是使用方自己约束，而不是下沉到框架这边约束，否则显得越俎代庖了

**Comment by kevwan** [comment]
> 返回的数据类型应该是通用能力吧？
> 
> 相应的规范应该是使用方自己约束，而不是下沉到框架这边约束，否则显得越俎代庖了

图灵完备框架？

**Reply by vcqr** [reply]
图灵完备只是通用编程的最低要求吧？
